commands.py

```diff
@@ -92,8 +99,38 @@ def get_players_count():
         server_player_count.port = server.port
         server_player_count.count = server.players.current
 
+        current_online_players_count += server.players.current
+        current_online_servers_count += 1
+
+        if server.players.current > 0:
+            current_active_servers_count += 1
+
         db.session.add(server_player_count)
 
+    click.echo('Getting peaks')
+
+    peak_refs = {
+        'total_players_peak_count': {'date_var_name': 'total_players_peak_date', 'current': current_total_players_count},
```

Comment:
Refactor this and use only one name. Append the prefix for the desired VAriable keys.